### ISSUE TITLE
Add support to cycle through multiple access tokens

### ISSUE THREAD

**Issue by techgaun** [techgaun/github-dorks#15]
Or maybe there's an alternative? But, if we can use tokens from multiple users, we can probably improve the time to wait.

**Comment by Plazmaz** [comment]
I'm working on a private project with similar functionality and we've had a lot of success with OAuth tokens.

**Comment by techgaun** [comment]
I think [Search API](https://developer.github.com/v3/search/#rate-limit) has custom rate limit which is same regardless of token type. The other way I can think of would be to just have the repo(s) cloned on the local for an org/person/repo and then run a scan locally. this would require rewriting the tool though

**Comment by Plazmaz** [comment]
@techgaun yes, however each independent key has a separate ratelimit. So if you had 2 or 3 friends, then you can achieve 2 or 3 times the search speed.

**Comment by techgaun** [comment]
ahh yeah thats what this issue is for :) I understood your comment differently (sorry). If you want to take this issue, I would be more than happy. Otherwise, I will try getting it done over next weekend.

**Comment by Plazmaz** [comment]
Unfortunately I don't know that I'll have the time for this. Also, my Python is very rusty.

**Comment by techgaun** [comment]
cool no problem
